doc/source/examples/03_modeling/chamfer.mystnb

```diff
@@ -0,0 +1,63 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Modeling: Chamfer edges and faces
+A chamfer is an angled cut on an edge. Chamfers can be created through the `Modeler.pull_tools` module.
```

Comment:
```suggestion
A chamfer is an angled cut on an edge. Chamfers can be created using the ``Modeler.pull_tools`` module.
```
Single backtick formats as italic. We need to use double backticks to format as a code entity.